russian/wca-regulations.md

```diff
@@ -1,19 +1,19 @@
 # <wca-title>Положения WCA
 
-<version>Версия: 1 мая 2019 г.
+<version>Версия: 1 января 2020 г.
 
 
 ## Замечания
 
 ### Перевод
 
-Это не официальные Положения WCA, а вольный перевод. В случае возникновения сомнений следует обращаться к оригинальным Положениям. В переводе возможны ошибки! Вопросы и замечания по поводу перевода отправляйте по электронной почте на [nick.masson@yandex.ru](mailto:nick.masson@yandex.ru).
+Это не официальные Положения WCA, а вольный перевод. В случае возникновения сомнений следует обращаться к оригинальным Положениям. В переводе возможны ошибки! Вопросы и замечания по поводу перевода отправляйте по электронной почте на [vihrianov1@outlook.com](mailto:vihrianov1@outlook.com).
```

Comment:
I think you shouldn't leave your own e-mail for contacts (at least for now). Let's leave it as it is now

Reply:
Ok.